experiments/functional/cspc-pool/block-deletion-with-volume/run_litmus_test.yml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: verify-cspc-pool-deletion-if-pool-with-volume-
+  namespace: litmus
+spec:
+  template:
+    metadata:
+      name: litmus
+      labels:
+        app: verify-cspc-pool-deletion-
+    spec:
+      serviceAccountName: litmus
+      restartPolicy: Never
+      containers:
+      - name: ansibletest
+        image: openebs/ansible-runner:ci
+        imagePullPolicy: IfNotPresent
+
+        env:
+          - name: ANSIBLE_STDOUT_CALLBACK
+            #value: log_plays, actionable, default
+            value: default
+
+           # Namespace where the OpenEBS components are deployed
+          - name: OPERATOR_NS
+            value: openebs
+
+          - name: APP_NAMESPACE
+            value: ""
+
+          - name: APP_LABEL
+            value: ""              
+
+        command: ["/bin/bash"]
+        args: ["-c", "ansible-playbook ./experiments/functional/cspc-pool/block-deletion-with-volume/test.yml -i /etc/ansible/hosts -v; exit 0"]
```

Comment:
change the directory name itself. Rename `block-deletion-with-volumes` to `pool-deletion-with-volumes`